Documentation/Input/HowToAddNearInteractivity.md

```diff
@@ -0,0 +1,148 @@
+# How to add near interaction in MRTK
+Near interactions come in the form of touches and grabs. Touch and grab events are raised as pointer events by the [PokePointer](Pointers.md#pokepointer) and [SpherePointer](Pointers.md#spherepointer), respectively.
+
+You can listen for touch and grab events by implementing [IMixedRealityPointerHandler](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler) and looking at the type of pointer that triggers your event. If the pointer is a PokePointer, the interaction is a touch. If the pointer is a SpherePointer, the interaction is a near grab.
+
+Because near interactions are dispatched by pointers, you need to make sure MRTK is configured to create a [SpherePointer](Pointers.md#spherepointer) and a [PokePointer](Pointers.md#pokepointer) whenever articulated hands are present. You also need to add [NearInteractionGrabbable](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionGrabbable) or [NearInteractionTouchable](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchable) to each collidable you want to be near grababable / touchable. If you want to add touch interactions to Unity UI, you need to add add [NearInteractionTouchableUnityUI](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchableUnityUI), read below for more details.
+
+For touch interaction it is also possible to configured your objects to raise touch events, listen to these by implementing the [IMixedRealityTouchHandler](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityTouchHandler)
```

Comment:
nit: to configured your objects => to configure objects
the comma and listen...also sounds weird. Either better transition or new sentence